runtime-kernel/linux-kernel-rpi64-lts/autobuild/build

```
@@ -0,0 +1,111 @@
+#
+# The Raspberry Pi build script
+# Copied from `linux-kernel' build.
+#
+abinfo "Preparing build environment ..."
+OLDARCH="$ARCH"
+unset ARCH
+if [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    SRCARCH="x86"
+elif [[ "${CROSS:-$ARCH}" = "arm64" ]]; then
+    SRCARCH="arm64"
+elif [[ "${CROSS:-$ARCH}" = "loongson3" ]]; then
+    SRCARCH="mips"
+elif [[ "${CROSS:-$ARCH}" = "ppc64el" ]]; then
+    SRCARCH="powerpc"    
+fi
+
+# RPi4 kernel build starts.
+
+abinfo "Building kernel for Raspberry Pi 4."
+
+abinfo "Copying config and setting up env..."
+
+cp -v autobuild/config-rpi4 "$SRCDIR"/.config
+
+. autobuild/build-common/set-var
+
+abinfo "Building kernel..."
+make
+
+abinfo "Stripping symbols..."
+. autobuild/build-common/strip-modules
+
+abinfo "Installing modules..."
+mkdir -p "$PKGDIR"/usr
+make INSTALL_MOD_PATH="$PKGDIR"/usr modules_install
+
+abinfo "Copying kernel..."
+# make a local copy of compiled kernel
+mkdir -p "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+mkdir -p "$PKGDIR/usr/lib/rpi64/kernel/overlays"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/rpi64/kernel/kernel8.img"
+
+export ARCH=$OLDARCH
+rm -rf "$PKGDIR"/usr/lib/firmware
+
+. autobuild/build-common/ext-hdr
+
+export LOCALNAME_RPI4=$LOCALNAME
+unset LOCALNAME
+
+# RPi4 kernel build ends.
+
+# Since `make mrproper` will destroy the installed kernel modules, we need to make a backup then.
+abinfo "Making backup for kernel and kernel modules..."
+tar -cvf rpi4-kernel.tar $PKGDIR
+
+abinfo "Removing all generated files + config + various backup files..."
+make mrproper
+
+# Now, restore the backup.
+abinfo "Restoring the backup..."
+tar -xvf rpi4-kernel.tar --directory=/
+
+# RPi5 kernel build starts.
+
+abinfo "Building kernel for Raspberry Pi 5."
+
+abinfo "Copying config and setting up env..."
+
+cp -v autobuild/config-rpi5 "$SRCDIR"/.config
+
+. autobuild/build-common/set-var
+
+abinfo "Building kernel..."
+make
+
+abinfo "Stripping symbols..."
+. autobuild/build-common/strip-modules
+
+abinfo "Installing modules..."
+mkdir -p "$PKGDIR"/usr
+make INSTALL_MOD_PATH="$PKGDIR"/usr modules_install
+
+abinfo "Copying kernel..."
+# make a local copy of compiled kernel
+mkdir -p "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+mkdir -p "$PKGDIR/usr/lib/rpi64/kernel/overlays"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/rpi64/kernel/kernel_2712.img"
+
+export ARCH=$OLDARCH
+rm -rf "$PKGDIR"/usr/lib/firmware
```

```suggestion
abinfo "Removing the unnecessary firmware in PKGDIR ..."
rm -rf "$PKGDIR"/usr/lib/firmware
```

A bit of output may help others to know what is going on :-)